test/functionalTest/harnessFunctions.sh

```diff
@@ -1239,7 +1244,6 @@ export -f brokerStart
 export -f localBrokerStop
 export -f localBrokerStart
 export -f brokerStop
-export -f localBrokerStop
```

Comment:
We call brokerStart and brokerStop from the functests, so those two must be exported (otherwise we cannot see them). Now, brokerStart calls localBrokerStart. It shouldn't be necessary to export localBrokerStart, IMHO.

I would do a test, exporting only brokerStart and brokerStop and if it works, then NOT export the other two.